Enable ccache in UDF examples build (#345)
NVIDIA/spark-rapids-examples: examples/UDF-Examples/RAPIDS-accelerated-UDFs/README.md

```bash
git clone https://github.com/NVIDIA/spark-rapids-examples.git
cd spark-rapids-examples/examples/UDF-Examples/RAPIDS-accelerated-UDFs
export LOCAL_CCACHE_DIR="$HOME/.ccache"
mkdir -p $LOCAL_CCACHE_DIR
export CCACHE_DIR="$LOCAL_CCACHE_DIR"
export CMAKE_C_COMPILER_LAUNCHER="ccache"
export CMAKE_CXX_COMPILER_LAUNCHER="ccache"
export CMAKE_CUDA_COMPILER_LAUNCHER="ccache"
export CMAKE_CXX_LINKER_LAUNCHER="ccache
mvn clean package -Pudf-native-examples
```

The Docker container has installed ccache 4.6 to accelerate the incremental building.

> You can change the LOCAL_CCACHE_DIR to a mounted folder so that the cache can be kept by next docker running.

Reviewer: ```suggestion
You can change the LOCAL_CCACHE_DIR to a mounted folder so that the cache can persist.
```
